src/main/java/mate/controllers/cars/AddingDriverToCarController.java

```diff
@@ -0,0 +1,33 @@
+package mate.controllers.cars;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddingDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private CarService carService = (CarService) injector.getInstance(CarService.class);
+    private DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/cars/add.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String carId = req.getParameter("carId");
+        String driverId = req.getParameter("driverId");
+        carService.addDriverToCar(driverService.get(Long.valueOf(driverId)),
+                carService.get(Long.valueOf(driverId)));
```

Comment:
```suggestion
        carService.addDriverToCar(driverService.get(Long.valueOf(driverId)),
                carService.get(Long.valueOf(carId)));
```
I guess it must be like this =)

Reply:
yes, your right))